### DISCUSSION TITLE
flattening by concatenation of property name

### DISCUSSION THREAD

**Discussion by samuelpsfung** [mapstruct/mapstruct#3101] [Q&A]
https://docs.automapper.org/en/latest/Flattening.html

> AutoMapper splits the destination member name into individual words (by PascalCase conventions).

`target.CustomerName = source.Customer.Name`

Can MapStruct do the same auto/global mapping?

**Comment by filiphr** [comment]
MapStruct doesn't provide such auto flattening.

However, you can configure it manually by doing:

```
@Mapping(target = "customerName", source = "customer.name")
```
